src/main/java/org/folio/services/group/GroupFiscalYearTotalsService.java

```diff
@@ -144,6 +144,7 @@ private GroupFiscalYearSummaryCollection convertHolders(List<GroupFiscalYearTran
 
   private void updateGroupSummaryWithAllocation(List<GroupFiscalYearTransactionsHolder> holders) {
     holders.forEach(holder -> {
+      holder.withToAllocations(new ArrayList<>(holder.getToAllocations()));
```

Comment:
Was there some bug with this previously, or you just added it to be safe if needed in future?

Reply:
I am not sure the bug could trigger before, but it certainly does now (changing an immutable list). I am not sure it is related to my changes.